acm/templates/policies/private-repo-policies.yaml

```diff
@@ -0,0 +1,161 @@
+# We copy the vp-private-repo-credentials from the "openshift-gitops" namespace
+# to the "open-cluster-management" via the "private-hub-policy"
+#
+# Then we copy the secret from the "open-cluster-management" namespace to the
+# managed clusters "openshift-gitops" instance
+#
+# And we also copy the same secret to the namespaced argo's namespace
```

Comment:
Oh wow.  But I also imagine we don't have any other choice

Reply:
Yeah, we could drive this imperatively via ansible, but that is worse than this imo